spec/index.bs

```diff
@@ -4962,54 +4951,59 @@ dictionary GPUExternalTextureDescriptor : GPUObjectDescriptorBase {
 
                 1. Let |source| be |descriptor|.{{GPUExternalTextureDescriptor/source}}.
 
-                1. If |source| <l spec=html>[=is not origin-clean=]</l>,
-                    throw a {{SecurityError}} and stop.
+                1. If the current image contents of |source| are the same as the most recent
+                    {{GPUDevice/importExternalTexture()}} call with an identical |descriptor|,
```

Comment:
> identical |descriptor|

Does that include the `label`? That feels strange, especially given some of the discussions we've had previously about shared internal objects (like bind group layouts).